packages/react-ui-core/src/Highlighter/__tests__/Highlighter-test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import Highlighter from '../Highlighter'
+
+describe('Highlighter', () => {
+  it('surounds pattern text with span and classname', () => {
+    const wrapper = mount(
+      <Highlighter pattern="test">
+        This is a test sentence for a test
+      </Highlighter>
+    )
+
+    wrapper.setProps({ pattern: 'test' })
+    expect(wrapper.find('[data-tid="highlighter-match"]')).toHaveLength(2)
+    const match = wrapper.find('[data-tid="highlighter-match"]').at(0)
+    expect(match.text()).toBe('test')
+    wrapper.setProps({ pattern: 'test2' })
+    expect(wrapper.find('[data-tid="highlighter-match"]')).toHaveLength(0)
+  })
+
+  it('surounds the nth pattern matched text with span and classname', () => {
+    const wrapper = mount(
+      <Highlighter pattern="test" indexHighlighted={2}>
+        This is a test sentence for a test
+      </Highlighter>
+    )
+
+    wrapper.setProps({ pattern: 'test' })
+    expect(wrapper.find('[data-tid="highlighter-match"]')).toHaveLength(1)
+    wrapper.setProps({ pattern: 'test2' })
+    expect(wrapper.find('[data-tid="highlighter-match"]')).toHaveLength(0)
```

Comment:
I set a const for  `wrapper.find('[data-tid="highlighter-match"]')`, but had to keep line 19 as is. I think it needs to run `find` again after the prop change on line 18

Reply:
Most likely, yes.